### ISSUE TITLE
ImGui Viewports (Multi-Windows)

### ISSUE THREAD

**Issue by Pastor111** [mellinoe/ImGui.NET#257]
Hi, i would like to ask if someone knows how to take advantage of the imgui viewports with imgui

**Comment by zaafar** [comment]
Checkout the docking branch.

**Comment by Pastor111** [comment]
Thank you for the suggestion, it almost worked but for some reason i get a System.AccesViolation

**Comment by zaafar** [comment]
i just clone the repository on a newly installed windows and changed to docking branch and compiled/executed the Sample Application that comes with ImGui.NET. Everything is working fine.

**Comment by benblo** [comment]
I'm confused by this... the master branch does have docking support (after enabling the ImGuiConfigFlags.DockingEnable flag, or via the demo window), but viewport support doesn't work for me. The docking branch has both working right out of the box without touching anything... so which branch is the preferred one? 
I know the docking branch has been the preferred one in vanilla imgui for years now, but if ImGui.NET supports docking in master, does it make any sense to have both branches anymore?

**Comment by mellinoe** [comment]
@benblo Right, there's no difference in the native bindings between the master branch and the docking branch anymore, and the version in the NuGet package supports both docking and multiple viewports by default. The only remaining difference is that the Veldrid renderer in the docking branch supports multiple viewports, and in the master branch it doesn't.

**Comment by benblo** [comment]
@mellinoe cool! I'm working off of the docking branch now, and everything is peachy. Please see PR #321 for more info.

**Comment by Vawlpe** [comment]
@mellinoe
> The only remaining difference is that the Veldrid renderer in the docking branch supports multiple viewports, and in the master branch it doesn't.

Heyyyy, I'm implement multiple viewports into a Veldrid renderer (partially) based on the version on `master` but I can't find all the `Platform_` delegates shown in the `docking` branch... anyone know where those would be located on `master`~? thx x3

**Comment by Vawlpe** [comment]
Alright so I figured I could just define my own delegates and it wouldn't be a big deal...
But then I realized `ImGuiNative.ImGuiPlatformIO_Set_Platform_GetWindow`(`Pos`/`Size`) are not exposed on `master`... fuck.

I couuuuld do the cimgui interop myself and define those 2 funcs but that feels very un-intended and yucky... any tips on what I could do instead~?
